pypeit/core/trace_slits.py

```diff
@@ -3039,6 +3039,9 @@ def trace_refine(filt_image, edges, edges_mask, ncoeff=5, npca = None, pca_expla
           Boost on fwhm for fweight.  This was 3.0 at one point (and that may be preferred for echelle instruments)
         maxrej: int, optional
           Rejection parameter for PCA.  1 makes the rejection go slowly (preferred)
+        smash_range: tuple, optional
```

Comment:
Please do not use rows/colums terminology as it is confusing in python. Use instead spatial and spectral direction to avoid ambiguity.

Reply:
Changed rows to spec